compile/compile_test.go

```diff
@@ -978,9 +978,7 @@ update {
 
 func modulesToString(modules []bundle.ModuleFile) string {
 	var buf bytes.Buffer
-	//result := make([]string, len(modules))
```

Comment:
This appears to be an unfinished cleanup from a few months back, so I took the liberty of completing the job. :sweat_smile: